src/manifest/clog.sidecar

```diff
@@ -15003,4 +15020,4 @@
       }
     ]
   }
-}
\ No newline at end of file
+}
```

Comment:
Did you add the trailing newline manually? Ideally we should leave this however the auto generation does.

Reply:
Actually, I've generated this with update-sidecar script, this is strange, I'll regenerate this